pkg/controllers/webhook/controller.go

```diff
@@ -876,7 +876,13 @@ func (c *controller) mergeWebhook(dst *webhook, policy kyvernov1.PolicyInterface
 		}
 	}
 	for _, gvr := range gvrsList {
-		dst.set(gvr)
+		if policy.GetNamespace() == "" {
+			// For ClusterPolices the Webhook rules scope should be "All Scopes"
+			dst.set(gvr, admissionregistrationv1.AllScopes)
+		} else {
+			// For Polices the Webhook rules scope should be "Namespaced"
+			dst.set(gvr, admissionregistrationv1.NamespacedScope)
+		}
```

Comment:
I don't think it's correct, a cluster policy can target namespaced resources.

Reply:
With the current implementation the `ClusterPolicy` would target namespaced and cluster scope resources.